src/wx/panel.cpp

```diff
@@ -2363,6 +2362,11 @@ void GLDrawingPanel::OnSize(wxSizeEvent& ev)
 {
     AdjustViewport();
 
+// Temporary hack to backport 800d6ed69b from wxWidgets until 3.2.2 is released.
+#if wxCHECK_VERSION(3, 2, 0) && !wxCHECK_VERSION(3, 2, 2)
+    MoveWaylandSubsurface(this);
```

Comment:
You can invoke this in every case and make it a no-op on other platforms (see other comments) - the linker will probably optimize it out, see other comments.